addons/account/views/account_invoice_view.xml

```diff
@@ -394,6 +394,7 @@
                                 <tree string="Invoice Lines" editable="bottom">
                                     <field name="sequence" widget="handle"/>
                                     <field name="product_id"/>
+                                    <field name="origin" invisible="1"/>
```

Comment:
@mart-e ,
Actually one client need this feature so we have added it in view and set invisible if he need this then he will make field Visible.